src/WebJobs.Script/Timer/AzureStorageScheduleMonitor.cs

```diff
@@ -0,0 +1,148 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.IO;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure;
+using Azure.Storage.Blobs;
+using Microsoft.Azure.WebJobs.Host.Executors;
+using Microsoft.Azure.WebJobs.Logging;
+using Microsoft.Azure.WebJobs.Script;
+using Microsoft.Extensions.Logging;
+using Newtonsoft.Json;
+
+namespace Microsoft.Azure.WebJobs.Extensions.Timers
+{
+    /// <summary>
+    /// This is an overriden implementation based off the StorageScheduleMonitor in Microsoft.Azure.WebJobs.Extensions package.
```

Comment:
In a remarks section here, can we label this TEMP to make it clear that this is a stop-gap solution until we release the actual storage v12 version of Microsoft.Azure.WebJobs.Host.Storage, when this will go away?

Same comment for any other types we've copied in temporarily.